### ISSUE TITLE
ModuleNotFoundError: No module named 'Bio.Seq'; 'Bio' is not a package

### ISSUE THREAD

**Issue by Darker1095** [MDAnalysis/mdanalysis#4317]
## Expected behavior ##

Convert xtc files to xyz files using mdanalysis

## Actual behavior ##
![image](https://github.com/MDAnalysis/mdanalysis/assets/104240838/6d10842e-4510-441c-97b4-c0ee206da34b)

File "/home/python3.7/site-packages/MDAnalysis/core/topologyattrs.py", line 45, in <module>
    import Bio.Seq
ModuleNotFoundError: No module named 'Bio.Seq'; 'Bio' is not a package

The error reported here is that I do not have the Bio module, but I am sure that I have biopython installed. I have tried to create a new environment directly using conda, and then installed it using ___conda install mdanalysis___. The same error is reported, and I am not sure where the problem is.

## Code to reproduce the behavior ##

``` python
import MDAnalysis as mda

xtc_file = 'nvt.xtc'
gro_file = '00data/nvt3_He.gro'

gro = mda.Universe(gro_file)
xtc = mda.Universe(xtc_file)

xtc.add_TopologyAttr('name', gro.atoms.names )
xtc.atoms.write('nvt.xyz', frames='all')

....

```

## Current version of MDAnalysis ##

- Which version are you using? (`MDAnalysis 2.6.1`)
- Which version of Python? (`python -3.11.6`)
- Which operating system?(`CentOS Linux release 7.4.1708`)

**Comment by IAlibay** [comment]
@Darker1095 could you provide the output of "conda list" here?

Also are you sure about your python and MDAnalysis versions? I'm seeing python 3.7 at the top of the issue and 3.10 at the bottom.

**Comment by IAlibay** [comment]
Ah ok, the second one is from your new environment? If you can provide the conda list output from that it would be grand.

**Comment by Darker1095** [comment]
OK. There is a problem with my previous statement because I tested it in multiple conda environments. Here I provide a "conda list" for installing the library through **conda install mdanalysis** in a new environment.

![image](https://github.com/MDAnalysis/mdanalysis/assets/104240838/1998b313-7f89-4ad2-94ad-5331a6cfe3d2)
![image](https://github.com/MDAnalysis/mdanalysis/assets/104240838/c04f11d3-2fd1-4a67-868c-439787da7cb5)
![image](https://github.com/MDAnalysis/mdanalysis/assets/104240838/c4c03bbd-f665-4520-b10e-e182f7305d13)

**Comment by Darker1095** [comment]
I have tried to change "import Bio.Seq" and "import Bio.SeqRecord" to "import  Bio" in the topologyattrs.py file in this environment(_anaconda3/envs/mda/lib/python3.11/site-packages/MDAnalysis/core/_)
it worked for me. But I don't understand what the problem was.

**Comment by orbeckst** [comment]
Is your conda environment Python 3.11? What was the command that you used to create your conda environment? This will help us to reproduce the issue.

Could you please edit your issue report to reflect the exact version numbers and insert a section on how you created the environment? Thanks.

**Comment by Darker1095** [comment]
@orbeckst Yes. I created the environment and installed mdanalysis using the following conda command
("conda config --add channels conda-forge","conda create -n mda","conda activate mda","conda install mdanalysis")

**Comment by orbeckst** [comment]
The screen shot in the issue comment indicates "python3.10" but you state "python -3.11.6". Which one is it?

Perhaps explicitly set the Python version in your env
```
conda config --add channels conda-forge
conda create -n mda310 python==3.10 mdanalysis mdanalysistests
conda activate mda310
```

Then run the tests (all in the same environment "mda310")
```
pytest --disable-pytest-warnings --pyargs MDAnalysisTests
```
to check that the installation is correct.

**Comment by Darker1095** [comment]
Thank you very much, my problem has been solved, although I don't know what happened
